WebHostLib/autolauncher.py

```diff
@@ -184,7 +188,12 @@ def start(self):
             return False
 
         process = multiprocessing.Process(group=None, target=run_server_process,
-                                          args=(self.name, self.ponyconfig, get_static_server_data(),
+                                          args=(self.name, self.ponyconfig,
+                                                self.room_port_min,
+                                                self.room_port_max,
+                                                self.room_port_overflow,
+                                                self.room_port_alloc_tries,
```

Comment:
this is quite messy. probably better way is to just pass in the whole config dict? Or group the related settings.